device: /dev/pcan32
baudrate: 500K
sync_interval: 10
chain_name: arm_controller
